mods/3d_armor/3d_armor/admin.lua

```diff
@@ -1,45 +0,0 @@
-minetest.register_alias("adminboots","3d_armor:boots_admin")
```

Comment:
Uhm, the admin armor should probably be incorporated again. Only @Darcidride uses it but it's still useful.

Reply:
It's registered in `armor.lua`.

Reply:
My bad!